Update README.rst

mwnelson opened the pull request:
Updated all instances of the institution name from either mkdriveninst or mktdriveninst to MktDrivenInst

mwnelson committed:
Update README.rst

Updated all instances of the institution name from either mkdriveninst or mktdriveninst to MktDrivenInst

katyhuff commented on README.rst:
```diff
@@ -1,4 +1,4 @@
-.. _mktdriveninst:
+.. _MktDrivenInst:
```
Hi @mwnelson ! This is the only change I'll request: instead of changing this line, just delete it.  This line isn't doing anything since I'm not wrapping the docs with sphinx.

(I could make this change to your pull request, but changes are typically suggested by the reviewer and actually made by the submitter. If you're in this for experience, we may as well do it the right way!)
You probably know this, but the way this works is : 
- you make a pull request (done!)
- I look at it and make comments on things that are awesome, questionable, or need to be changed (done!)
- You make changes accordingly to your patch-1 branch and push those commits to mwnelson/patch-1 (that will automagically update the pull request here)
- I pull patch-1 once it satisfies all my desires

mwnelson replied:
Ok great, thanks for playing along to assist me in learning the ropes!

On Tuesday, August 26, 2014 4:27 PM, Katy Huff notifications@github.com wrote:

In README.rst:

> @@ -1,4 +1,4 @@
> -.. _mktdriveninst:
> +.. _MktDrivenInst: 
> Hi @mwnelson ! This is the only change I'll request: instead of changing this line, just delete it.  This line isn't doing anything since I'm not wrapping the docs with sphinx.
> (I make this change to your pull request, but changes are typically suggested by the reviewer, but made by the submitter. If you're in this for experience, we may as well do it the right way!)
> You probably know this, but the way this works is : 
>     \* you make a pull request (done!)
>     \* I look at it and make comments on things that are awesome, questionable, or need to be changed (done!)
>     \* You make changes accordingly to your patch-1 branch and push those commits to mwnelson/patch-1 (that will automagically update the pull request here)
>     \* I pull patch-1 once it satisfies all my desires
> —
> Reply to this email directly or view it on GitHub.

mwnelson committed:
Update README.rst

AHHH! Everything is new to me! Ought I delete both empty lines? Neither? Should I have already seen the answer to this question somewhere in the linked Google style guides? Does it matter?

katyhuff commented:
congratulations @mwnelson !
